packages/semver/src/executors/version/index.e2e.spec.ts

```diff
@@ -1085,6 +1086,40 @@ $`)
       ).toEqual('0.0.1-beta.0');
     });
 
+
+    it(`should bump "a"'s package.json`, async () => {
+      expect(
+        (await lastValueFrom(readPackageJson('packages/a'))).version
+      ).toEqual('0.0.1-beta.0');
+    });
+
+
+    it(`should bump "a"'s package.json with different preids`, async () => {
+      await version(
+        {
+          ...defaultBuilderOptions,
+          version: 'prerelease',
+          syncVersions: true,
+          preid: 'alpha',
+        },
+       fakeContext
+      );
+      await version(
+        {
+          ...defaultBuilderOptions,
+          syncVersions: true,
+          version: 'prerelease',
+          preid: 'alpha',
+        },
+       fakeContext
+      );
+
+      expect(
+        (await lastValueFrom(readPackageJson('packages/a'))).version
+      ).toEqual('0.0.1-alpha.1');
+    });
```

Comment:
I would suggest using custom --preid for these new tests, like the one you mentioned in the issue (`add-feature`, `fix-bug`) and also moving them out in another describe function.